cmd/influx/main.go

```diff
@@ -158,7 +158,10 @@ Examples:
 	}
 
 	if err := c.connect(""); err != nil {
-
```

Comment:
what???  lol, crazy!  I think code reviews failed us the last time.  Nice catch.